### DISCUSSION TITLE
How to reset Admin Panel Password/Email?

### DISCUSSION THREAD

**Discussion by FelixNiedermann** [pocketbase/pocketbase#2654] [Q&A] [answered]
I've got everything running on a VPS, with Nginx proxy, I can access the frontend and admin panel but admin panels ask for login not for registering, I downloaded the new linux pocketbase file uploaded it, but still the same. How can I reset or how is it done on a server that I can create a panel account?

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand what do you mean. Usually the first time when you start PocketBase with blank or missing `pb_data` you'll be asked to create your first admin account. I guess something with the nginx proxy is interfering with the initial setup redirect? In any case you can try to manually access the install url at `https://yourdomain.com/_/?installer` OR use the helper `admin` console command to create/update your account:

```sh
# create
./pocketbase admin create test@example.com 1234567890

# update
./pocketbase admin update test@example.com 1234567890

# delete
./pocketbase admin delete test@example.com
```

**Reply by FelixNiedermann** [reply]
Yeah that's what I get normally I excepted also on the server to happen...
I don't have a pb_data (deleted that for that reason)
Accessing it with installer will redirect me to login
And console command give me the following:
root@ubuntu:/home/Projects/Pocketbase_Test# ./pocketbase admin create test@example.com 1234567890
bash: ./pocketbase: Permission denied
<img width="163" alt="Bildschirmfoto 2023-06-06 um 13 21 03" src="https://github.com/pocketbase/pocketbase/assets/45990943/990e4009-02e7-4fd4-8204-f16296e1f03a">

**Reply by ganigeorgiev** [reply]
I'm confused. The `pb_data` directory should be created automatically on `./pocketbase serve` when missing. If you don't have one, then I'm not sure how are you even able to start the executable.

The permission error is even more strange as from your example it seems that you are running the command with root. Are you even able to start the executable with `./pocketbase serve`?

If you are able to start the executable with blank `pb_data` and the installer page is redirecting you to the login then I guess it is possible to be caused by some misconfiguration of the reverse proxy.

Could you provide a little more details about your environment - PocketBase version, nginx config, etc.?

**Reply by FelixNiedermann** [reply]
I made an error, I was running it with the command itself not the systemd, restarted the command and everything works for that, but now I tried stopping the command and let it run as systemd service and that doesn't work now, not sure why as the ExexStart command is exactly the same as I put manually in the terminal and which worked:
[Unit]
Description = pocketbase

[Service]
Type           = simple
User           = root
Group          = root
LimitNOFILE    = 4096
Restart        = always
RestartSec     = 5s
StandardOutput = append:/root/pb/errors.log
StandardError  = append:/root/pb/errors.log
WorkingDirectory = /home/Projects/Pocketbase_Test
ExecStart = /home/Projects/Pocketbase_Test/pocketbase serve --http="backend.xy.ch:8090"

[Install]
WantedBy = multi-user.target

**Reply by ganigeorgiev** [reply, marked as answer]
Does the stdout and stderr dir path (aka. `/root/pb/`) exist?

If not, try to create them or change the directories path (eg. somewhere inside your `/home/Projects/Pocketbase_Test`)

**Reply by FelixNiedermann** [reply]
And my Nginx:
server {
    listen 80;
    server_name backend.xy.ch;

    location /test {
        alias /home/Projects/Pocketbase_Test/pb_public/;
        index app.html;
    }

    location /test/_ {
        proxy_pass http://backend.xy.ch:8090;  # not 100% what to put here, tried a few things
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
        rewrite /test/(.*) /$1  break;
    }

    location /test/api {
        proxy_pass http://backend.xy.ch:8090;  # not 100% what to put here, tried a few things
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
        rewrite /test/(.*) /$1  break;
    }

    location / {
        alias /home/Projects/Main/;
        index index.html;
    }
}

But this worked when I executed the serve command myself

**Reply by ganigeorgiev** [reply]
Please check the above comment about the `StandardOutput` and `StandardError` paths in your systemd service file.

About the nginx config - do you really need 3 separate location blocks? I haven't tested it but a single `location /test/` should be enough in your case.

**Reply by FelixNiedermann** [reply]
Ah this was it...the folders didn't existed thank you
